### PR TITLE
Improve the structure of this repository (resolves #3)

### LINKED ISSUE #3: Pushes into the data repo master branch triggers Netlify deploys
## Is your feature request related to a problem? Please describe.

Accessibility map uses data from this repo. The push of new commits into the master branch should trigger Netlify to re-deploy.

## Describe the solution you'd like

Use Github actions to listen to pushes into the master branch. When a push occurs, send a POST request to the Netlify build hook to trigger the re-deploy.

### PR DESCRIPTION
**Pull request by cindyli**

### CODE REVIEW

**Comment by cindyli** [comment]
@amb26 and I talked about a simple way to get the data change in this data repository reflected on the map site. That is to directly access the URL of `./latest.csv` in this repository, rather than redeploying the map site everytime.

This PR is updated to reflect this change. The directory structure is also changed to hold: 1. datasets from ODS; 2. merged datasets. Right now, all files are in `/ODS_datasets` although one or two files are no longer original ODS files. They are placed in there because they are all waiting for a merge.

This PR is ready for a review.

**Review by TeddTech — APPROVED** [review]

**Comment by amb26** [comment]
The file you have renamed as assessment_centre_locations_2020_09_02.csv is not an ODS dataset - it is @ptcher 's working copy in which he is adding fresh columns and working notes. It should go into a different directory. I don't think the current split, of mixing in upstream data and working data in the same directory makes sense - also there is nothing special about a "merged dataset", it is just a dataset that belongs to some particularly community which has resulted from a merge - for example, once we merge the latest ODS into @ptcher 's previous working copy, that merged result will then become his new working copy. I suggest we instead organise data by ownership/provenance or schema, so rather than "ODS" and "merged", we should have "ODS", "WeCount", and perhaps others (accessibility cloud, etc.). I realise that we are going to get into problems pretty quickly with our "materialised view" where we have each version of the file listed out in the repo - it will make it impossible, once we come to merge files, to use git to track the lineage of them - unless we start writing complex bits of metadata out in JSON to list what the "parents" of a particular merged file are. That said, we might well want to do this, just so as we don't run into other problems trying to synthesise git commits that pretend to be from particular times in the past.
Let's imagine that we will indeed stick small .json files with lineage metadata alongside the CSV files, and go with a draft plan where we organise directories by ownership. Instead of one overall "latest.csv" we will have a separate "latest.csv" in each directory, and alongside it we can write a little JSON "latest.json" which for now will just hold contents like:
````
{ "branch" : "assessment_centre_locations_2020_08_06.csv"}
````
We could also consider doing away with latest.csv entirely and just store the JSON file, since it looks like we aren't going to be able to directly host the files out of github to the client anyway, because of the cross-site problem you reported.

**Comment by cindyli** [comment]
@amb26, thanks for the review. This PR is ready for another look.

File: latest.json
Comment (amb26) [inline latest.json]: I think each directory should have its own separate "latest.json" file. If we have any metadata at this level (I'm not sure we need any at this stage), it should refer to full file paths from its own point of view e.g. "ODS/assessment_centre_locations_2020_09_22.csv" rather than expecting the reader to reconstruct the convention.

Comment (cindyli) [reply latest.json]: @amb26, if each directory has its own "latest.json" file, is it still necessary to specify the file path such as "ODS/..." in it? Having the path in `latest.json` or a schema file would make sense when a file or files are not hardcopied to the community directory but being linked in from elsewhere. We haven't talked about this case.

If not considering to have the file path info, what do you think of making `latest.json` look like:
```
{
	"fileName": "assessment_centre_locations_2020_09_22.csv"
}
```

Comment (amb26) [reply latest.json]: Yes, that's correct, I think the filenames in latest.json should be relative to the directory holding it

**Comment by cindyli** [comment]
@amb26, thanks for the clarification. The review comment has been addressed. This PR is ready for another review.

**Comment by amb26** [comment]
One last tweak - I think we should retain the originator's original name for the WeCount spreadsheets, which was assessment_centre_data_collection_2020_09_02.csv

**Comment by cindyli** [comment]
@amb26, the last comment has been addressed. Ready for another look.

**Comment by amb26** [comment]
I guess we need to think about acquiring some kind of "CSV lint" tool for such respositories - @the-t-in-rtf, @jobara perhaps something like https://www.npmjs.com/package/csvlint could be integrated into our lint-all task?

**Comment by the-t-in-rtf** [comment]
We'd have to write our own task, but it's possible.  It'd be nice to discuss how to cleanly distinguish which checks to run at all, so that people can still run "lint-all" and opt-in/out of checks via configuration.

**Pull request merged**
